contracts/base/ModuleManager.sol

```diff
@@ -152,11 +161,10 @@ abstract contract ModuleManager is SelfAuthorized, Executor, IModuleManager {
         uint256 value,
         bytes memory data,
         Enum.Operation operation
-    ) public virtual override returns (bool success) {
+    ) public override returns (bool success) {
         (address guard, bytes32 guardHash) = preModuleExecution(to, value, data, operation);
-
         success = execute(to, value, data, operation, type(uint256).max);
-        postModuleExecution(guard, guardHash, success);
+        postModuleExecution(to, value, data, operation, guard, guardHash, success);
```

Comment:
I wonder if:


```suggestion
        postModuleExecution(guard, guardHash, success);
        onAfterExecTransactionFromModule(to, value, data, operation, success);
```

Uses slightly less gas. I imagine the code size would be a smudge bigger, but maybe it is slightly more gas optimal.


Don't need to push the change, but might be worth playing around and seeing if moving the code has any effects.

Reply:
#### This PR

Safe 21813 bytes (limit is 24576)
SafeL2 22631 bytes (limit is 24576)

```
·------------------------------------------------------------|----------------------------|-------------|------------------------------·
|                    Solc version: 0.7.6                     ·  Optimizer enabled: false  ·  Runs: 200  ·  Block limit: 100000000 gas  │
·····························································|····························|·············|·······························
|  Methods                                                                                                                             │
·····················|·······································|·············|··············|·············|···············|···············
|  Safe              ·  execTransactionFromModule            ·      51650  ·      184255  ·     127887  ·           11  ·           -  │
·····················|·······································|·············|··············|·············|···············|···············
|  Safe              ·  execTransactionFromModuleReturnData  ·      52293  ·      183999  ·     107939  ·            5  ·           -  │

```

#### After moving code

https://github.com/safe-global/safe-smart-account/commit/d1ae12d9a4ff9c86f626f9319c309659dd9b2d14

Safe 21814 bytes (limit is 24576)
SafeL2 22632 bytes (limit is 24576)

```
·------------------------------------------------------------|----------------------------|-------------|------------------------------·
|                    Solc version: 0.7.6                     ·  Optimizer enabled: false  ·  Runs: 200  ·  Block limit: 100000000 gas  │
·····························································|····························|·············|·······························
|  Methods                                                                                                                             │
·····················|·······································|·············|··············|·············|···············|···············
|  Safe              ·  execTransactionFromModule            ·      51630  ·      184235  ·     127867  ·           11  ·           -  │
·····················|·······································|·············|··············|·············|···············|···············
|  Safe              ·  execTransactionFromModuleReturnData  ·      52273  ·      183979  ·     107919  ·            5  ·           -  │
```